test/system/helpers.bash

```diff
@@ -379,6 +379,10 @@ function is_netavark() {
     return 1
 }
 
+function is_aarch64() {
+    [ "$(uname -m)" -eq "aarch64" ]
```

Comment:
In shell languages `-eq` is for numbers and `=` for strings. (I know. Please don't blame me.) Suggestin:
```bash
    [[ "$(uname -m)" == "aarch64" ]]
```
And, you actually have to use the new `skip_if_aarch64` code somewhere :)

Reply:
lol yes, i realized after pushing the image ID change that I had this diff too. I'll check out the failures and then add this.